### ISSUE TITLE
Fix for Potree webgl crash on recent Chrome browsers which default to webGL2 and have no FragDepth extension

### ISSUE THREAD

**Issue by justgord** [potree/potree#1393]
**TLDR**

Recent Chrome defaults to webGL2 .. breaking shaders which need FragDepth as an extension
proposed fix/workaround is to explicitly request webGL1 from THREE.js
  

**To Reproduce**

git clone current potree repo
npm install
npm start
visit : http://localhost:1234/examples/lion.html using recent Chrome v 120.0.6099.109 on linux
lion shows okay in default square point mode
click point cloud, change point display to Paraboloid
webgl crashes, with console error :

            Uncaught could not compile shader pointcloud: WARNING: 0:12: 'GL_EXT_frag_depth' : extension is not supported
            ERROR: 0:81: 'gl_FragDepthEXT' : undeclared identifier

Error also happens with many other Potree features, which depend on frag depth

**Explanation**

recent chrome defaults to webGL2, which does not have a FragDepth extension, it is built in
many features of potree have shaders which request fragdepth
rather than change all those, a minimal workaround with less code changes, is to ask threejs to request webGLv1 

**Proposed Fix : request WebGL1 via Three js**

In src/viewer/viewer.js Viewer.initThree() : 
request THREE.WebGL1Renderer  to get webgl1 context
and just prior to that call, do not request a default webglcontext, which may be webgl2

        let canvas = document.createElement("canvas");

        this.renderer = new THREE.WebGL1Renderer({ // WebGL1 for FragDepth extension
            alpha: true,
            premultipliedAlpha: false,
            canvas: canvas});

  Code diff attached, in lieu of pull request
[potree_fix_diff.txt](https://github.com/potree/potree/files/13733888/potree_fix_diff.txt)

**Note**

This workaround will likely require a lot of regression tests, which I dont have resources for.
Essentially the current 1.8 code base seems reliant on webGL1, so making this explicit is probably a good idea, as more browsers default to webGL2 in future.

**Comment by ukarthik27** [comment]
Is there a dynamic way to do this? Check if webgl has `GL_EXT_frag_depth` extension and then downgrade to webgl1renderer during initialization process ?
